ui/app/registration/views/patientcommon.html

```diff
@@ -278,6 +278,24 @@
         <div patient-relationship patient="patient" ng-show="patient.hasRelationships"
              ng-if="::relationshipTypes"></div>
     </div>
+    <div class="box-container">
+        <legend class="additional-attribute" toggle="patient.hasIndexInfo">
+          <span>
+              <i class="fa fa-caret-right"></i>
+              <i class="fa fa-caret-down"></i>
+              <strong><a ng-bind-html="::'REGISTRATION_TITLE_INDEX_TESTING_CONTACT_INFO' | translate"></a></strong>
+          </span>
+        </legend>
+
+        <div ng-show="patient.hasIndexInfo" ng-repeat="attributeRow in ::patientConfiguration.customAttributeRows()">
+            <article style="margin: 0" class="form-field" ng-repeat="attribute in ::attributeRow"
+                ng-class="::{'right-form-field' : $index%2 != 0}" ng-if="attribute.name == 'PERSON_ATTRIBUTE_TYPE_INDEX_KNOW_VIH_STATUS'">
+                <attribute-types target-model="patient" attribute="::attribute" field-validation="::fieldValidation"
```

Comment:
@tmvumbi2 Fetching attributes like this only shows the ones that are not hidden. And if they are not hidden, they will show in the Other Information section. Please check the death information section to investigate ways of fetching the attributes in some other way.